mitoinstaller/mitoinstaller/installer_steps/final_installer_steps.py

```diff
@@ -25,6 +24,11 @@ def replace_process_with_jupyter():
     """
     if is_running_test():
         return
+
+    if ('--turn-off-jupyter-launch' in sys.argv):
```

Comment:
Can we name this `--no-jupyter-launch`? Turn off makes it seem like it _sets_ something, rather than just not launching this time.

Otherwise, LGTM.